packages/lit-dev-cloudbuild-cleanup/src/cleanup.ts

```diff
@@ -0,0 +1,272 @@
+import {google} from 'googleapis';
+import {Octokit} from 'octokit';
+import {request} from 'gaxios';
+
+const ONE_WEEK_IN_MS = 7 * 24 * 60 * 60 * 1000;
+const ONE_WEEK_AGO = new Date(Date.now() - ONE_WEEK_IN_MS);
+
+const {PROJECT_ID, REPO_NAME, _DEPLOY_REGION} = process.env;
+
+if ([PROJECT_ID, REPO_NAME, _DEPLOY_REGION].some((env) => env === undefined)) {
+  console.log({PROJECT_ID, REPO_NAME, _DEPLOY_REGION});
+  throw new Error(
+    'missing one or more required environment variables: PROJECT_ID, REPO_NAME, _DEPLOY_REGION'
+  );
+}
+
+const sleep = (time: number) =>
+  new Promise((resolve) => setTimeout(resolve, time));
+
+async function main() {
+  const octokit = new Octokit();
+
+  const openPrs = new Set<number>();
+
+  console.log('fetching open PRs from github');
+  const openPrsIterator = octokit.paginate.iterator(octokit.rest.pulls.list, {
+    owner: 'lit',
+    repo: `${REPO_NAME}`,
```

Comment:
```suggestion
    repo: REPO_NAME,
```

Reply:
Due to the way I'm checking it, and maybe also because it's inside an async function(?), TS still thinks `REPO_NAME` could be undefined so it didn't match the param type without the template literal/conversion to string. Ah, I suppose I could do `REPO_NAME as string`.

Reply:
Ah yeah, the `${}` is actually masking the type error there, since that does implicit `undefined` string conversion. Would prefer something explicit.